devices/owon.js

```diff
@@ -121,6 +122,11 @@ module.exports = [
             await reporting.instantaneousDemand(endpoint, {min: 5, max: constants.repInterval.MINUTES_5, change: 2}); // divider 1000: 2W
             await reporting.currentSummDelivered(endpoint, {min: 5, max: constants.repInterval.MINUTES_5,
                 change: [10, 10]}); // divider 1000: 0,01kWh
+
+            if (device.powerSource === 'Unknown') {
+                device.powerSource = 'Mains (single phase)';
```

Comment:
No need for this if check, for a power plug we can assume it is always main powered.

Reply:
I have just pushed a new commit that removes the check